modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/UpNextQueue.kt

```diff
@@ -115,3 +158,20 @@ fun Observable<UpNextQueue.State>.containsUuid(uuid: String): Observable<Boolean
         }
     }
 }
+
+fun <T, R> Flow<T>.switchMap(mapper: (value: T) -> Flow<R>): Flow<R> {
```

Comment:
I don't think you need to introduce a new operator. `flatMapLatest()` acts effectively the same. And if you need to introduce a new operator it should be in a different module with tests.